m02_setup/01_first_steps.rst

```diff
@@ -0,0 +1,190 @@
+
+The web pages of A+ courses are written in `RST <http://docutils.sourceforge.net/docs/user/rst/quickref.html>`_. RST is simpler in syntax than HTML. A Python software called `Sphinx <http://www.sphinx-doc.org>`_ compiles a bunch of RST files into HTML. However, we will explain more about :doc:`RST and Sphinx </m03_rst_guide/01_get_started>` later.
+
+.. admonition:: Epigraph :glyphicon-certificate:`\ `
+  :class: meta
+
+  In the start there was A+. That was good, but a developer thought we needed something more. Thus, MOOC-Grader was created.
+
+  Time passed, and now we have many microservices that provide our new way of life.
+
+  -- A+ LMS
+
+If you are already running the course template on your computer, and seeing the content in a web browser, *congratulations!* You can skip the rest of this page. 
+
+Knowledge prerequisites
+-----------------------
+
+This manual assumes you have some basic UNIX operating system skills. It also assumes that you can use GNU/Linux (Ubuntu, Debian) or modern Mac OS, and assumes that you know some basic :abbr:`Unix terminal (The terminal is a text-based window where you type commands, and the terminal shows the results by printing one or more lines of text.)` commands. You also need to know how to install some software in your computer. 
+
+In this course, we will teach you how to install and how to use the software.
+
+.. image:: /images/terminal.png
+  :width: 70%
+  :align: center
+
+.. admonition:: Software: UNIX Shell, Terminal emulator
+  :class: alert alert-info
+
+  The above image list the content of the **aplus-manual**
+  directory, in a terminal emulator window. In practice, some terminal
+  commands are required for the usual workflow of writing
+  a new course for A+.
+
+  The terminal is used for moving and copying files, starting software
+  and examining their error messages. To be precise, there is the terminal
+  emulator window and inside it another program, `shell`, which reads
+  user commands and acts according to them. If you are new to this, read the
+  `"Learning the shell" LinuxCommand.org <http://linuxcommand.org/lc3_learning_the_shell.php>`_
+  tutorial parts 1-6.
+
+Software prerequisites
+----------------------
+
+The first step to start authoring a course is to install some required software on your computer. The first two software you need are `git <https://git-scm.com/>`_ and `make <https://www.gnu.org/software/make/>`_. Sometimes, these two software are installed by default, but if the **git** and **make** are not installed by default, you can install them by running the :ref:`commands below <install-git>` in your machine. Otherwise, you can consult the official documentation on how to `install git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_, and how to `install make <https://pkgs.org/download/build-essential>`_ in your OS. 
```

Comment:
This text claims that teachers must install `make`, but that is incorrect. RST text is compiled in a container and thus, `make` is not used in the native host under normal circumstances. It would be needed if someone had to compile without the container, but we don't recommend that.